Makefile

```diff
@@ -199,4 +211,30 @@ publish-helm: ## Helm publish
 	helm push ./kro-${RELEASE_VERSION}.tgz oci://${HELM_IMAGE}
 
 .PHONY:
-release: build-image publish-image package-helm publish-helm
\ No newline at end of file
+release: build-image publish-image package-helm publish-helm
+
+##@ Deployment
+
+ifndef ignore-not-found
+  ignore-not-found = true
+endif
+
+.PHONY: install
+install: manifests ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+	$(KUBECTL) apply -f ./helm/crds
+
+.PHONY: uninstall
+uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config
+	$(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f ./helm/crds
+
+.PHONY: deploy-kind
+deploy-kind:
+	$(KIND) delete clusters ${KIND_CLUSTER_NAME} || true
+	$(KIND) create cluster --name ${KIND_CLUSTER_NAME}
+	$(KUBECTL) create namespace kro-system
+	make install
+	make build-image # TODO: how do you kind.local here ?
```

Comment:
re:TODO, don't build the image like this here, do it on the line 238: `helm template | kubectl apply` line, change `kubectl apply` to `ko apply`

The whole point of using ko in this way with kind is to use the node image loading feature of ko and not need a secondary registry. Sure you can get it to work other ways too and really ko does not care, just set the correct `KO_DOCKER_REPO` and you are gtg. 

Another reason why I don't like this kind of stuff inside the make file, it is too complex to expose all the ways you might want to do this.

Reply:
I wasn’t sure how that works. Will try that and update it here.